src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color 0.3s, transform 0.3s;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(1.2, 1.2);
+    }
   }
 
   &__card:not(:nth-child(4n)) {
```

Comment:
[SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.